docs/static/playground/pxt-common/pxt-core.d.js

```diff
@@ -387,6 +387,18 @@ declare interface String {
 //% text.defl="123"
 declare function parseFloat(text: string): number;
 
+/**
+ * Returns a pseudorandom number between min and max included.
+ * If both numbers are integral, the result is integral.
+ * @param min the lower inclusive bound, eg: 0
+ * @param max the upper inclusive bound, eg: 10
+ */
+//% blockId="device_random" block="pick random %min|to %limit"
+//% blockNamespace="Math"
+//% help=math/randint
+//% shim=Math_::randomRange
+declare function randint(min: number, max: number): number;
```

Comment:
I forget how the playground works... is  this a build artifact or does it need to be kept in sync with pxt-common?

Reply:
Needs to be in sync. I'll update it